### PR TITLE
Made WFC Panel accessible to admin team & changed filename of WFC export

### PR DESCRIPTION
**Pull request by danieljames-dj**

### CODE REVIEW

File: WcaOnRails/app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class StaticPagesController < ApplicationController
 
   before_action :current_user_is_authorized_for_action!, only: [:panel_wfc]
   private def current_user_is_authorized_for_action!
```
Comment (gregorbg) [inline WcaOnRails/app/controllers/static_pages_controller.rb:7]: This method name tells you nothing about the fact that this authorization is based on WFC.

**Review by gregorbg — CHANGES_REQUESTED** [review]
I'm a bit reluctant about the complexity of the permission changes.
Usually, we go about this by introducing a `def can_do_something?` method in `user.rb` and referencing that "permission method" in a `redirect_to_root_unless_user` hook.

I think to avoid code duplication and to keep all of our permissions in one place, you should modify `user.rb` in this case as well and remove all new (direct!) WFC-related permission checks in the controller (of course, the controller still has to do the permission checks but it should do so by delegating them to the user, similar to how other permission checks are handled in this very controller already.)

File: WcaOnRails/app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (danieljames-dj) [reply WcaOnRails/app/controllers/static_pages_controller.rb]: Okay changed.

**Comment by danieljames-dj** [comment]
> I'm a bit reluctant about the complexity of the permission changes. Usually, we go about this by introducing a `def can_do_something?` method in `user.rb` and referencing that "permission method" in a `redirect_to_root_unless_user` hook.
> 
> I think to avoid code duplication and to keep all of our permissions in one place, you should modify `user.rb` in this case as well and remove all new (direct!) WFC-related permission checks in the controller (of course, the controller still has to do the permission checks but it should do so by delegating them to the user, similar to how other permission checks are handled in this very controller already.)

Done, I've added function in user.rb and used it to check permission.

File: WcaOnRails/app/models/user.rb [hunk 2]
```diff
@@ -1251,4 +1251,8 @@ def senior_or_self
   def is_delegate_in_probation
     Role.where(user_id: self.id).where("end_date is null or end_date >= curdate()").present?
   end
+
+  def can_view_wfc_matters?
```
Comment (gregorbg) [inline WcaOnRails/app/models/user.rb:1255]: There is already `user.can_admin_finances?` which I believe can be reused for this PR

**Review by gregorbg — CHANGES_REQUESTED** [review]

**Review by gregorbg — APPROVED** [review]

**Pull request merged**